### DISCUSSION TITLE
Embed instead of iFrame

### DISCUSSION THREAD

**Discussion by yonnath** [Alex-D/Trumbowyg#1420] [Q&A]
Hi everyone,

How can I paste a vimeo video URL and get the corresponding code to display using an embed instead of an iFrame?

The use case is: I'm measuring the time spent on a webpage which contains videos and it's almost impossible to check activity within the iframe, whereas it's possible in an embed, and displaying vimeo videos is possible using both iFrame and Embed tags, but the paste embed plugin brings an iframe tag, not an embed tag.

Thanks in advance for any ideas or hints… :-)

Anthony

**Comment by Alex-D** [comment]
Which plugin are you using?

I am pretty sure that you can customize the code if you want :)

**Comment by yonnath** [comment]
Salut Alex,

Thanks for your attention! 🙏🏻
I'm using the **Paste embed** plugin to insert videos from **Vimeo** in a website built with HTML5/bootstrap/PHP.

Have a nice day
